packages/alfa-rules/test/sia-r75/rule.spec.tsx

```diff
@@ -1,70 +1,81 @@
-import { h } from "@siteimprove/alfa-dom";
+import { Declaration, h, Node } from "@siteimprove/alfa-dom";
+import { Option } from "@siteimprove/alfa-option";
 import { test } from "@siteimprove/alfa-test";
 
 import R75, { Outcomes } from "../../src/sia-r75/rule";
 
 import { evaluate } from "../common/evaluate";
 import { passed, failed, inapplicable } from "../common/outcome";
 
+const fontSize = (value: string) => Declaration.of("font-size", value);
+const makeTarget = (
+  name: string,
+  children: Array<string | Node>,
+  declaration: Declaration
+) => h.element(name, [], children, [declaration]);
+
 test("evaluate() passes an element with a font size not smaller than 9 pixels", async (t) => {
-  const target = <html style={{ fontSize: "medium" }}>Hello world</html>;
+  const declaration = fontSize("medium");
+  const target = makeTarget("html", ["Hello World"], declaration);
 
   const document = h.document([target]);
 
   t.deepEqual(await evaluate(R75, { document }), [
     passed(R75, target, {
-      1: Outcomes.IsSufficient,
+      1: Outcomes.IsSufficient(Option.of(declaration)),
     }),
   ]);
 });
 
 test("evaluate() fails an element with a font size smaller than 9 pixels", async (t) => {
-  const target = <div style={{ fontSize: "8px" }}>Hello world</div>;
+  const declaration = fontSize("8px");
+  const target = makeTarget("div", ["Hello World"], declaration);
 
   const document = h.document([target]);
 
   t.deepEqual(await evaluate(R75, { document }), [
     failed(R75, target, {
-      1: Outcomes.IsInsufficient,
+      1: Outcomes.IsInsufficient(Option.of(declaration)),
     }),
   ]);
 });
 
 test(`evaluate() fails an element with an accumulated font size smaller than 9
       pixels`, async (t) => {
-  const target1 = <p style={{ fontSize: "smaller" }}>Hello world</p>;
-  const target2 = <div style={{ fontSize: "10px" }}>{target1}</div>;
+  const declaration1 = fontSize("smaller");
+  const target1 = makeTarget("p", ["Hello World"], declaration1);
+
+  const declaration2 = fontSize("10px");
+  const target2 = makeTarget("div", [target1], declaration2);
 
   const document = h.document([target2]);
 
   t.deepEqual(await evaluate(R75, { document }), [
     passed(R75, target2, {
-      1: Outcomes.IsSufficient,
+      1: Outcomes.IsSufficient(Option.of(declaration2)),
     }),
     failed(R75, target1, {
-      1: Outcomes.IsInsufficient,
+      1: Outcomes.IsInsufficient(Option.of(declaration1)),
     }),
   ]);
 });
 
 test(`evaluate() passes an element with too small a font size when the font size
```

Comment:
Being nitpick: `evaluate() passes an element with a too small font size when the font size
      does not affect descendant text`